docs/Deploying-Pega-on-PKS.md

```diff
@@ -82,7 +84,7 @@ During deployment the required Kubernetes configuration file is copied into the
 
 ### Creating a database resource
 
-PKS deployments require you to install Pega Platform software in an SQL database. After you create an SQL instance that is available to your PKS cluster, you must create a PostgreSQL database in which to install Pega Platform. When you are finished, you will need the database name and the SQL instance IP address which you create in this procedure in order to add this information to your pega.yaml Helm chart.
+PKS deployments require you to install Pega Platform software in an PostgreSQL database. After you create an SQL instance that is available to your PKS cluster, you must create a PostgreSQL database in which to install Pega Platform. When you are finished, you will need the database name and the SQL instance IP address which you create in this procedure in order to add this information to your pega.yaml Helm chart.
```

Comment:
Not PostgreSQL specific.  If PKS is on AKS, you may want to use MSSQL.

Reply:
ooopsey...good catch!